### PR TITLE
Prettier products

### PR DESCRIPTION
**Pull request by Marmak95**
Improved modal to view some of the information about the product when clicking on a picture of the product in the home page and if the stock is more than 0, then the "buy" button will appear to take the user to the product information page.
Pages were added, but doesn't take the user to another page yet.

### CODE REVIEW

File: templates/product_info.html [hunk 1]
```diff
@@ -55,31 +55,70 @@
...
+            </figure>
+            <div class="media-content">
+                <div class="content">
+                    <a href="/product/info/{{ item.serial_number }}" style="color:black;">
+                        <div class="content">
+                            <h3>
+                                <strong >{{ item.name }}</strong>
+                            </h3>
+                            <nav class="level is-mobile">
+                                <div class="level-left">
```
Comment (ivajon) [inline templates/product_info.html]: Remove this level

File: templates/jinja_macros/product_listing.html [hunk 2]
```diff
@@ -10,47 +10,127 @@
...
+                                    max-width:12em;
+                                    height:auto;
+                                    width:auto">
+                    </figure>
+                    <div class="media-content">
+                        <div class="content">
+                            <div class="column">
+                                <div class="content">
+                                    <div class="level is-mobile">
+                                        <!-- These should be fetched from the db -->
```
Comment (ivajon) [inline templates/jinja_macros/product_listing.html]: Remove this level until we actually fetch the attributes from db

**Review by ivajon — COMMENTED** [review]
Looks good aside from the 2 comments

File: templates/product_info.html (hunk 1 above)
Comment (Marmak95) [reply templates/product_info.html]: The level has been removed.

File: templates/jinja_macros/product_listing.html (hunk 2 above)
Comment (Marmak95) [reply templates/jinja_macros/product_listing.html]: The level has been removed.

**Comment by ivajon** [comment]
Looks good to me

**Pull request merged**